clang/lib/AST/MicrosoftMangle.cpp

```diff
@@ -2986,6 +2989,9 @@ void MicrosoftCXXNameMangler::mangleCallingConvention(CallingConv CC) {
     case CC_Swift: Out << 'S'; break;
     case CC_SwiftAsync: Out << 'W'; break;
     case CC_PreserveMost: Out << 'U'; break;
+    case CC_PreserveNone:
+      Out << 'V';
```

Comment:
This is not a good long term solution. I emailed the relevant external Microsoft mailing list for discussing these issues and CC'd some folks to inquire about a better long term solution, but we can continue with this approach for now.